src/tools/rbd_mirror/InstanceWatcher.cc

```diff
@@ -294,7 +294,7 @@ void InstanceWatcher<I>::get_instances(librados::IoCtx &io_ctx,
   C_GetInstances *ctx = new C_GetInstances(instance_ids, on_finish);
   librados::AioCompletion *aio_comp = create_rados_callback(ctx);
 
-  int r = io_ctx.aio_operate(RBD_MIRROR_LEADER, aio_comp, &op, &ctx->out_bl);
+  int r = io_ctx.aio_operate(RBD_MIRROR_LEADER /* XXXMG */, aio_comp, &op, &ctx->out_bl);
```

Comment:
Although we don't create per namespace leader watcher, we still use "rbd_mirror_leader" object for operations like this one. It seems we could use "rbd_mirror" object for this, but this would break compatibility with older rbd-mirror daemons. Not sure, if we want to do something with this.